addons/mass_mailing/migrations/13.0.2.0/pre-migration.py

```diff
@@ -41,7 +41,6 @@
 
 _field_renames = [
     ('link.tracker.click', 'link_tracker_click', 'mail_stat_id', 'mailing_trace_id'),
-    ('mailing.trace', 'mailing_trace', 'mass_mailing_campaign_id', 'campaign_id'),
```

Comment:
Why is this incorrect? `campaign_id` doesn't exist before v13, so the renaming is good for me.

Reply:
The problem here is you are renaming a field `mass_mailing_campaign_id` (of model `mail.mass_mailing.campaign`) to field campaign_id (of model `utm.campaign`), which obviously that's wrong.